salt/journal-cms/init.sls

```diff
@@ -148,7 +148,18 @@ site-settings:
 journal-cms-{{ key }}:
     mysql_database.present:
         - name: {{ db.name }}
+        {% if salt['elife.cfg']('cfn.outputs.RDSHost') %}
+        # remote mysql
+        - connection_user: {{ salt['elife.cfg']('project.rds_username') }} # rds 'owner' uname
+        - connection_pass: {{ pillar.elife.db_root.password }} # rds 'owner' pass
```

Comment:
it's not even tested yet! there are still syntax errors! I pushed what I had to a separate branch. I'll be working on this code this morning. relax.